### PR TITLE
[Feature-2] Second feature in this project

### PR DESCRIPTION
**Pull request by bijanipiran**
Just added new feature:

-  New folder created
- New file was added into the folder

### CODE REVIEW

File: feature-2/another-feature.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file added in feature-2 branch
+
```
Comment (bijanipiran) [inline feature-2/another-feature.txt:2]: It is good idea to write this line

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (bijanipiran) [inline .gitignore:1]: you ignore .idea

**Review by bijanipiran — COMMENTED** [review]
comment

File: feature-2/another-feature.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+This file added in feature-2 branch
```
Comment (mohammadbijani) [inline feature-2/another-feature.txt:1]: good

**Review by mohammadbijani — COMMENTED** [review]
that is good

**Pull request merged**